src/main/java/com/scalefocus/blog_api/request/BlogUpdateRequest.java

```diff
@@ -0,0 +1,4 @@
+package com.scalefocus.blog_api.request;
+
+public record BlogUpdateRequest(String title,String text) {
```

Comment:
using Reuqest class for update is generally good idea, congratz on thinking in this way. But just be careful in cases when this can lead to boilerplate code and the complexity of maintaining two almost similar objects.